compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -88,10 +87,20 @@ buildDar service pkgConf@PackageConfigFields {..} ifDir dalfInput = do
         then liftIO $
              Just <$> do
                  bytes <- BSL.readFile pMain
-                 createArchive pkgConf "" bytes [] [] [] []
+                 createArchive pkgConf "" bytes [] (toNormalizedFilePath ".") [] [] []
         else runAction service $
              runMaybeT $ do
                  WhnfPackage pkg <- useE GeneratePackage file
+                 parsedMain <- useE GetParsedModule file
```

Comment:
That looks really similar to [`moduleImportPaths`](https://github.com/digital-asset/daml/blob/b87215e111a0a99209d3993b9e527551c178cace/compiler/hie-core/src/Development/IDE/GHC/Util.hs#L97). Can we reuse the code? If not, why not?

Reply:
sure, I'll check what's in `moduleImportPaths`. I'll extend the integration tests a bit in my next PR, we're also missing one for the `merge` command.